element_deeplabcut/model.py

```diff
@@ -98,19 +91,16 @@ def get_dlc_root_data_dir() -> list:
 
 
 def get_dlc_processed_data_dir() -> str:
-    """
-    If specified by the user, this function provides DeepLabCut with an output
-    directory for processed files. If unspecified, output files will be stored
-    in the session directory 'videos' folder, per DeepLabCut default
+    """Pulls relevant func from parent namespace. Defaults to DLC's project /videos/.
 
-    get_dlc_processed_data_dir -> str
-        This user-provided function specifies where DeepLabCut output files
-        will be stored.
+    Method in parent namespace should provide a string to a directory where DLC output
+    files will be stored. If unspecified, output files will be stored in the
+    session directory 'videos' folder, per DeepLabCut default.
     """
     if hasattr(_linking_module, "get_dlc_processed_data_dir"):
         return _linking_module.get_dlc_processed_data_dir()

```

Comment:
The default should be to return to the root_data_dir in case the dlc_processed is not defined

Reply:
My motivation for changing was that, if I had 2 roots and my project was in the second, the previous logic inferred the output as the first root, not finding the actual project. I'm hoping to avoid that bug.

It's not clear to me how it's backwards incompatible if we're also changing the logic of how we manage the value below. If not provided, I'm finding the proper root instead of defaulting to the first.